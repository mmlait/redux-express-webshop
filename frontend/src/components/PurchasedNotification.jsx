import React from 'react';
import styled from 'styled-components';
import LightHeading from './general/styledComponents';
import colors from '../colors';

const ModalWrapper = styled.div`
  background-color: ${colors.modalBgOverlay};
  position: fixed;
  z-index: 1;
  padding-top: 100px;
  left: 0;
  top: 0;
  width: 100%;
  height: 100%;
`

const NotificationContent = styled.div`
  background-color: ${colors.primary};
  display: flex;
  flex-direction: column;
  justify-content: center;
  margin: auto;
  width: 90%;
  height: 200px;
  border-radius: 15px;
  @media (min-width: 450px) {
    max-width: 450px;
  }
  @media (min-width: 900px) {
    width: 500px;
    height: 250px;
  }
`

const PurchasedNotification = () => {
  return (
    <ModalWrapper>
      <NotificationContent>
        <LightHeading>Thank you for purchasing!</LightHeading>
      </NotificationContent>
    </ModalWrapper>
  );
}

export default PurchasedNotification;