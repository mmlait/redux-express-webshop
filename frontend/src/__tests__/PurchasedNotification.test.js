import React from 'react';
import PurchasedNotification from '../components/PurchasedNotification';
import { create } from 'react-test-renderer'

describe('PurchasedNotification', () => {
    it('should render PurchasedNotification component', () => {
    let tree = create(<PurchasedNotification />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})