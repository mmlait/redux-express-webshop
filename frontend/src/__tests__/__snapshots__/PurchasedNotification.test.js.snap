// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PurchasedNotification should render PurchasedNotification component 1`] = `
<div
  className="sc-bxivhb cWBDwX"
>
  <div
    className="sc-ifAKCX eBLFKN"
  >
    <h2
      className="sc-bdVaJa emjykE"
    >
      Thank you for purchasing!
    </h2>
  </div>
</div>
`;
